projects/dot-rules/src/lib/semantic/modules/dropdown/dropdown.ts

```diff
@@ -1,163 +1,133 @@
+import { MenuItem } from 'primeng/components/common/api';
```

Comment:
maybe we should take advantage of this change to add doc to all this component

Reply:
https://github.com/dotCMS/core-web/pull/955#discussion_r265733261